src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/MetricsEmitter.java

```diff
@@ -810,4 +810,199 @@ public static void emitNodeMetrics(
           "Total time taken for writing {} metrics metricsdb: {}", tableName, mFinalT - mCurrT);
     }
   }
+
+  public static void emitShardStateMetric(
```

Comment:
It might be good to add a javadoc comment with a sample metric emitted in the writer files.

Reply:
Added